chapters/th/_toctree.yml

```
@@ -6,4 +6,20 @@
 - title: 1. โมเดล Transformer
   sections:
   - local: chapter1/1
-    title: บทนำ
\ No newline at end of file
+    title: บทนำ
+
+- title: 4. การแบ่งปันโมเดลและ tokenizers
+  sections:
+  - local: chapter4/1
+    title: Hugging Face Hub
```

Small suggestion to keep the title the same as the English counterpart

```suggestion
    title: The Hugging Face Hub
```